site/build.js

```diff
@@ -52,7 +52,16 @@ metalSmith(__dirname)
     .use(function (files, metalsmith, next) {
         var metadata = metalsmith.metadata();
 
-        var assertionsByType = {};
+        // Make sure that the most important types are listed first and in this order:
+        var assertionsByType = {
+            any: [],
+            array: [],
+            boolean: [],
+            function: [],
+            number: [],
+            object: [],
+            string: []
+        };
```

Comment:
Yes, it would, if you really think that sorting the types alphabetically is good enough. I found it a bit weird that `function` and `object` were thrown in between the primitives, so I figured it was just a matter of time before we would want to maintain a more logical ordering.